api/src/routes/identity/index.ts

```diff
@@ -62,6 +69,13 @@ export class IdentityRoutes {
 	addUser = async (req: Request, res: Response, next: NextFunction): Promise<any> => {
 		try {
 			const user = req.body as IdentitySchemaBody;
+
```

Comment:
I think we should write and capture exception instead of doing read/write.
Read and insert can lead to race conditions, while the simple createIdentity is atomic.

Reply:
See comment above. I would make it consistent with createidentity